tls/s2n_handshake.h

```diff
@@ -53,6 +53,13 @@ struct s2n_handshake {
     struct s2n_hash_state sha256;
     struct s2n_hash_state sha384;
     struct s2n_hash_state sha512;
+    struct s2n_hash_state md5_sha1;
+
+    /* Used for SSLv3 */
+    struct s2n_hash_state sslv3_md5_copy;
```

Comment:
These are used to copy the handshake state's hash being used to a copy workspace used by the PRF. I've added "PRF" to those comments for clarity.